ci/deps/azure-windows-37.yaml

```diff
@@ -8,14 +8,14 @@ dependencies:
   - dateutil
   - gcsfs
   - html5lib
-  - jinja2=2.8
+  - jinja2
   - lxml
-  - matplotlib=2.0.1
+  - matplotlib=3.0.1
```

Comment:
Kept fixed MPL pin to have more version coverage (as it was before for 2.0.1). There's a floating MPL version somewhere already.